### PR TITLE
[ticket/13154] Event to edit user list for notification in notifications

### PR DESCRIPTION
**Pull request by brunoais**
https://tracker.phpbb.com/browse/PHPBB3-13154

PHPBB3-13154

### CODE REVIEW

File: tests/notification/base.php [hunk 1]
```diff
@@ -66,6 +66,8 @@ protected function setUp()
 			$phpbb_root_path,
 			$phpEx
 		);
+		
+		$phpbb_dispatcher = $this->phpbb_dispatcher = new phpbb_mock_event_dispatcher();
```
Comment (nickvergessen) [inline tests/notification/base.php]: You don't need `$phpbb_dispatcher` so don't assign it?

Comment (brunoais) [inline tests/notification/base.php]: I did it to make it the same way as all other are doing. Should I still not assign it?

Comment (nickvergessen) [inline tests/notification/base.php]: Its a defined variable that is not needed, yes please remove it

Comment (nickvergessen) [inline tests/notification/base.php]: The others are needed due to `global $db, $config, $user, $auth, $cache, $phpbb_container;`

Comment (brunoais) [inline tests/notification/base.php]: I also added it to global, though...

Comment (nickvergessen) [inline tests/notification/base.php]: Not on this PR?!

Comment (brunoais) [inline tests/notification/base.php]: Look down @ submit_post_base.php

Comment (nickvergessen) [inline tests/notification/base.php]: submit_post_base.php itself creates `$phpbb_dispatcher = new phpbb_mock_event_dispatcher();`

**Pull request merged**